src/components/Book.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ book }) => (
+  <tr>
+    <td>{book.id}</td>
+    <td>{book.title}</td>
+    <td>{book.category}</td>
+  </tr>
+);
```

Comment:
**REQUIRED**
* It accepts `handleRemoveBook` prop.
* It adds Remove book button to the book row [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/RemoveBookButton.png) from path right
* It attaches passed handleRemoveBook to on click event of Remove book button